src/os/bluestore/BlueFS.cc

```diff
@@ -1993,6 +1995,9 @@ int BlueFS::_allocate(uint8_t id, uint64_t len,
         to_release.insert(p.offset, p.length);
       }
       alloc[id]->release(to_release);
+
+      if(logger && (id != BDEV_SLOW))
+        logger->inc(l_bluefs_fragment_fallback, 1);
```

Comment:
Perhaps it's a bit better to increment this below (under if(bdev[id]) statement) along with logging corresponding notification.

Reply:
Shouldn't we additionally track total amount that was spilled over to slow device? I.e introduce one more counter incremented by left param here.

Reply:
Shouldn't we also distinguish fallback to DB device from the one to SLOW device? I.e. introduce yet another pair of counters?

Reply:
maybe the `slow_used_bytes`  and `db_used_bytes` already enough for the space watch purpose?

Reply:
@Aran85 - these parameters track total amount of space allocated at the corresponding volumes. Including both data originally targeted to them and spilled over data. But they don't allow to track just the spilled over amounts, i.e. amounts of data that had to reside at faster volumes but finally has landed at slower ones due to the lack of space.